zetaclient/logs/fields.go

```diff
@@ -3,27 +3,36 @@ package logs
 // A group of predefined field keys and module names for zetaclient logs
 const (
```

Comment:
In the end I'm thinking we could document the fields by commenting the variable in this file, in the future we want a proper ZetaClient on the website, we can dynamically get the documentation by generated docs from this file